packages/edit-site/src/components/sidebar-navigation-item/style.scss

```diff
@@ -22,17 +22,6 @@
 		fill: $gray-700;
 	}
 
-	&:is(a) {
-		text-decoration: none;
-		display: flex;
-		align-items: center;
-
-		&:focus {
-			box-shadow: none;
-			outline: none;
-		}
```

Comment:
This focus style seems to be derived from WP-Admin's `common.css`.

![common](https://github.com/WordPress/gutenberg/assets/54422211/82f721d9-ba98-41a7-b261-466f2c2b1920)


I think it would be better to resolve this at the component level, so I will move this focus style to the component.